10-jre-aba/Dockerfile

```diff
@@ -0,0 +1,94 @@
+    && GCC_LIBS_URL="https://archive.archlinux.org/packages/g/gcc-libs/gcc-libs-8.2.1%2B20180831-1-x86_64.pkg.tar.xz" \
+    && GCC_LIBS_SHA256=e4b39fb1f5957c5aab5c2ce0c46e03d30426f3b94b9992b009d417ff2d56af4d \
+    && ZLIB_URL="https://archive.archlinux.org/packages/z/zlib/zlib-1%3A1.2.9-1-x86_64.pkg.tar.xz" \
+    && ZLIB_SHA256=bb0959c08c1735de27abf01440a6f8a17c5c51e61c3b4c707e988c906d3b7f67 \
+    && curl -Ls https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub -o /etc/apk/keys/sgerrand.rsa.pub \
+    && curl -Ls ${ALPINE_GLIBC_REPO}/${GLIBC_VER}/glibc-${GLIBC_VER}.apk > /tmp/${GLIBC_VER}.apk \
+    && apk add /tmp/${GLIBC_VER}.apk \
+    && curl -Ls ${GCC_LIBS_URL} -o /tmp/gcc-libs.tar.xz \
+    && echo "${GCC_LIBS_SHA256}  /tmp/gcc-libs.tar.xz" | sha256sum -c - \
+    && mkdir /tmp/gcc \
+    && tar -xf /tmp/gcc-libs.tar.xz -C /tmp/gcc \
+    && mv /tmp/gcc/usr/lib/libgcc* /tmp/gcc/usr/lib/libstdc++* /usr/glibc-compat/lib \
+    && strip /usr/glibc-compat/lib/libgcc_s.so.* /usr/glibc-compat/lib/libstdc++.so* \
+    && curl -Ls ${ZLIB_URL} -o /tmp/libz.tar.xz \
+    && echo "${ZLIB_SHA256}  /tmp/libz.tar.xz" | sha256sum -c - \
+    && mkdir /tmp/libz \
+    && tar -xf /tmp/libz.tar.xz -C /tmp/libz \
+    && mv /tmp/libz/usr/lib/libz.so* /usr/glibc-compat/lib \
+    && apk del --purge .build-deps \
+    && rm -rf /tmp/${GLIBC_VER}.apk /tmp/gcc /tmp/gcc-libs.tar.xz /tmp/libz /tmp/libz.tar.xz /var/cache/apk/*
+
+ENV JAVA_VERSION jdk10u
+
+COPY slim-java* /usr/local/bin/
+
+RUN set -eux; \
+    apk add --virtual .fetch-deps curl; \
+    ARCH="$(apk --print-arch)"; \
+    case "${ARCH}" in \
+       ppc64el|ppc64le) \
+         ESUM='e529ed62a88c944b247a1e0bc522a7ce613ba1e4b6df45219acfd4ddc7af7f03'; \
+         BINARY_URL='https://github.com/AdoptOpenJDK/openjdk10-binaries/releases/download/jdk10u-2018-09-29-10-32/OpenJDK10U_ppc64le_linux_openj9_2018-09-29-10-32.tar.gz'; \
+         ;; \
+       s390x) \
+         ESUM='7a364d9d5c4c0452f4f50c23ac24bf1a3eb5ee52733f0f46c1214947533d114f'; \
+         BINARY_URL='https://github.com/AdoptOpenJDK/openjdk10-binaries/releases/download/jdk10u-2018-09-26-18-22/OpenJDK10U_s390x_linux_openj9_2018-09-26-18-22.tar.gz'; \
+         ;; \
+       amd64|x86_64) \
+         ESUM='e980fbe5f71fdf66c368f4c88efebd191df0cb7114f3702b6ab83976ca994554'; \
+         BINARY_URL='https://github.com/AdoptOpenJDK/openjdk10-binaries/releases/download/jdk10u-2018-09-29-10-32/OpenJDK10U_x64_linux_openj9_2018-09-29-10-32.tar.gz'; \
+         ;; \
+       *) \
+         echo "Unsupported arch: ${ARCH}"; \
+         exit 1; \
+         ;; \
+    esac; \
+    curl -Lso /tmp/openjdk.tar.gz ${BINARY_URL}; \
+    sha256sum /tmp/openjdk.tar.gz; \
+    mkdir -p /opt/java/openjdk; \
+    cd /opt/java/openjdk; \
+    echo "${ESUM}  /tmp/openjdk.tar.gz" | sha256sum -c -; \
+    tar -xf /tmp/openjdk.tar.gz; \
+    jdir=$(dirname $(dirname $(find /opt/java/openjdk -name javac))); \
+
```

Comment:
El docker build se está quejando de todos los espacios en blanco de este comando RUN:
[WARNING]: Empty continuation line found in:
    RUN set -eux;     apk add --virtual .fetch-deps curl;     ARCH="$(apk --print-arch)";     case "${ARCH}" in        ppc64el|ppc64le)          ESUM='e529ed62a88c944b247a1e0bc522a7ce613ba1e4b6df45219acfd4ddc7af7f03';          BINARY_URL='https://github.com/AdoptOpenJDK/openjdk10-binaries/releases/download/jdk10u-2018-09-29-10-32/OpenJDK10U_ppc64le_linux_openj9_2018-09-29-10-32.tar.gz';          ;;        s390x)          ESUM='7a364d9d5c4c0452f4f50c23ac24bf1a3eb5ee52733f0f46c1214947533d114f';          BINARY_URL='https://github.com/AdoptOpenJDK/openjdk10-binaries/releases/download/jdk10u-2018-09-26-18-22/OpenJDK10U_s390x_linux_openj9_2018-09-26-18-22.tar.gz';          ;;        amd64|x86_64)          ESUM='e980fbe5f71fdf66c368f4c88efebd191df0cb7114f3702b6ab83976ca994554';          BINARY_URL='https://github.com/AdoptOpenJDK/openjdk10-binaries/releases/download/jdk10u-2018-09-29-10-32/OpenJDK10U_x64_linux_openj9_2018-09-29-10-32.tar.gz';          ;;        *)          echo "Unsupported arch: ${ARCH}";          exit 1;          ;;     esac;     curl -Lso /tmp/openjdk.tar.gz ${BINARY_URL};     sha256sum /tmp/openjdk.tar.gz;     mkdir -p /opt/java/openjdk;     cd /opt/java/openjdk;     echo "${ESUM}  /tmp/openjdk.tar.gz" | sha256sum -c -;     tar -xf /tmp/openjdk.tar.gz;     jdir=$(dirname $(dirname $(find /opt/java/openjdk -name javac)));     export PATH="/opt/java/openjdk/bin:$PATH";     export PATH="${jdir}/bin:$PATH";     apk add --virtual .build-deps bash binutils;     /usr/local/bin/slim-java.sh ${jdir}/bin;     ${jdir}/bin/jlink --compress=2 --module-path /opt/java/openjdk/jmods --add-modules openj9.sharedclasses,java.base,java.logging,java.xml,jdk.unsupported,java.sql,java.naming,java.desktop,java.management,java.security.jgss,java.instrument --output /linked;     mv /linked/* /opt/java/openjdk;     apk del --purge .build-deps;     apk del --purge .fetch-deps;     rm -rf /var/cache/apk/*;     rm -rf ${jdir} /tmp/openjdk.tar.gz;     rm -rf /linked;
[WARNING]: Empty continuation lines will become errors in a future release.

Reply:
done